payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -367,11 +363,7 @@ object PaymentsTheme {
         @ReadOnlyComposable
         get() = typographyMutable.toComposeTypography()
 
-    var primaryButtonModifierMutable = PaymentsThemeDefaults.primaryButtonModifier
-    val primaryButtonModifier: PrimaryButtonModifierCompose
-        @Composable
-        @ReadOnlyComposable
-        get() = primaryButtonModifierMutable.toComposeModifier(isSystemInDarkTheme())
+    var primaryButtonMutable = PaymentsThemeDefaults.primaryButtonTheme
```

Comment:
Is "mutable" still needed on this variable name?

Reply:
nope. I missed this when refactoring.